frontend/src/metabase/dashboard/components/ParametersPopover.tsx

```diff
@@ -63,20 +53,21 @@ const ParameterOptionsSection = ({
   section: ParameterSection;
   onClick: () => void;
 }) => (
-  <OptionItemRoot onClick={onClick}>
-    <OptionItemTitle
-      className={cx(CS.textBold, CS.flex, CS.alignCenter)}
-      style={{ marginBottom: 4 }}
-    >
+  <Box className={S.row} display="table-row" onClick={onClick}>
```

Comment:
:x: Please use accessible controls.

We can use Mantine's [Menu](https://v6.mantine.dev/core/menu/).

Reply:
Fixed